django/contrib/admin/templates/admin/change_list_results.html

```diff
@@ -18,7 +18,7 @@
          <a href="{{ header.url_toggle }}" class="toggle {{ header.ascending|yesno:'ascending,descending' }}" title="{% translate "Toggle sorting" %}"></a>
        </div>
    {% endif %}
-   <div class="text">{% if header.sortable %}<a role="button" href="{{ header.url_primary }}">{{ header.text|capfirst }}</a>{% else %}<span>{{ header.text|capfirst }}</span>{% endif %}</div>
+   <div class="text" id="{{ header.id }}">{% if header.sortable %}<a role="button" href="{{ header.url_primary }}">{{ header.text|capfirst }}</a>{% else %}<span>{{ header.text|capfirst }}</span>{% endif %}</div>
```

Comment:
Wondering if / how we should handle a case where `id` isn’t defined. Is that possible? If so I assume we would not want to add the `id=` attribute to the HTML tag?